### PR TITLE
feat: Added action to upload multiple files in S3

### LINKED ISSUE #6737: [Feature] Multiple file uploads are very messy
## Description

Multiple file uploading needs a job queue in frontend and might needs to be separated from single file upload. As it might need to load another JS queue library. Can't see what files were successfully uploaded

### Steps to reproduce the behaviour:

TBD

### Important Details

- Version: [Cloud / Self-Hosted vx.x]
- OS: [e.g.MacOSX]
- Environment [production, release, deploy preview]

### PR DESCRIPTION
**Pull request by nidhi-nair**
Fixes #6737

### CODE REVIEW

**Review by sumitsum — CHANGES_REQUESTED** [review]

File: app/server/appsmith-plugins/amazons3Plugin/src/main/java/com/external/plugins/constants/AmazonS3Action.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 public enum AmazonS3Action {
     LIST,
     UPLOAD_FILE_FROM_BODY,
+    UPLOAD_MULTIPLE_FILES_FROM_BODY,
```
[the remark this answers is not in the document]
Comment (nidhi-nair) [reply app/server/appsmith-plugins/amazons3Plugin/src/main/java/com/external/plugins/constants/AmazonS3Action.java:6]: From our list of UQI methods that we want plugins to support, CRUD and multi CRUD are separated out into separate methods, which is why I created a separate action.

**Review by sumitsum — APPROVED** [review]

**Pull request merged**